cvat-ui/src/components/annotation-page/standard-workspace/objects-side-bar/issues-list.tsx

```diff
@@ -39,7 +39,11 @@ export default function LabelsListComponent(): JSX.Element {
     const ready = useSelector((state: CombinedState) => state.annotation.canvas.ready);
     const activeControl = useSelector((state: CombinedState) => state.annotation.canvas.activeControl);
 
-    let frames = issues.map((issue: Issue): number => issue.frame).sort((a: number, b: number) => +a - +b);
+    let frames = issues
+        .filter((issue: Issue) => !issue.resolved) // Filter out the resolved issues
```

Comment:
The filter should only be applied if the hide issues setting enabled

Reply:
alright